src/styles.css

```
@@ -736,3 +736,30 @@ button.mat-button[_ngcontent-c12] {
+  z-index: 3;
+}
+.mat-add-btn:disabled {
+  opacity: 0.5;
+}
+.mat-remove-btn {
+  background-color:#C21F1F;
+  min-width: 0;
+  border-radius: 20px;
+  border-color: unset !important;
+  color: white;
+  border: unset;
```

Biome (1.9.4)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)